pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -565,7 +565,7 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 		// If the ordinal could not be parsed (ord < 0), ignore the Pod.
 	}
 
-	// for any empty indices in the sequence [0,set.Spec.Replicas) create a new Pod at the correct revision
+	// for any empty indices in the sequence [start,end] create a new Pod at the correct revision
```

Comment:
Good catch, thanks @shreyas44 for the changes.

Reply:
@shreyas44 but seems like a very trivial change can you provide more changes.

Reply:
don't have anything more to contribute right now. was going through the codebase to understand controllers and noticed this and created a pr to fix it.

Reply:
You can find more errors or bugs you can find and improve the file.